ground/src/main/java/com/google/android/ground/repository/LocationOfInterestRepository.kt

```diff
@@ -125,6 +125,9 @@ constructor(
   fun getLocationsOfInterestOnceAndStream(survey: Survey): Flowable<Set<LocationOfInterest>> =
     localLoiStore.getLocationsOfInterestOnceAndStream(survey)
 
+  suspend fun getAllGeometries(survey: Survey): List<Geometry> =
```

Comment:
Don't we already load all the features when opening a survey? Would we be doing it twice instead of once after this chance?

Reply:
Thanks for flagging this! Fixed to read the LOIs from local database.